lib/core/presentation/widgets/common/app_backdrop/app_backdrop.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:ui';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+class AppBackdrop extends StatelessWidget {
+  const AppBackdrop({
+    Key? key,
+    this.strength = 1,
+    this.child,
+  }) : super(key: key);
+
+  final double strength;
+  final Widget? child;
+
+  @override
+  Widget build(BuildContext context) {
+    final double normalStrength = clampDouble(strength, 0, 1);
```

Comment:
Add this to force maximum strength

Refer from : https://github.com/gskinnerTeam/flutter-wonderous-app/blob/main/lib/ui/common/app_backdrop.dart